shared/fs/banner/container.js

```diff
@@ -37,12 +37,16 @@ const mergeProps = (stateProps, dispatchProps, {path}: OwnProps) => ({
   path,
 })
 
-export default compose(
-  connect(mapStateToProps, mapDispatchToProps, mergeProps),
-  setDisplayName('FilesBanner'),
-  lifecycle({
-    componentDidMount() {
-      this.props.getFuseStatus()
-    },
-  })
-)(Files)
+const ConnectedBanner = isMobile
```

Comment:
For this change, [w=1](https://github.com/keybase/client/pull/11416/commits/af9af8d1d95d8799b5a1ce5ddf0c523764f9b595?w=1) is your friend!